### PR TITLE
[GSP-22] Create API documentation

### PR DESCRIPTION
**Pull request by ktrof**

### CODE REVIEW

File: backend/src/main/java/com/gridstudentpractice/chatservice/Application.java [hunk 1]
```diff
@@ -2,14 +2,45 @@
...
+        return new ApiInfo(
+                "Chatservice API",
+                "API for Grid Dynamics student practice",
+                "1.0",
+                "Free to use",
+                new Contact("Mantarus", "https://github.com/Mantarus", "https://github.com/Mantarus"),
+                "API License",
+                "https://github.com/Mantarus/grid-student-practice",
+                Collections.emptyList());
     }
```
Comment (Mantarus) [inline backend/src/main/java/com/gridstudentpractice/chatservice/Application.java]: I would prefer to move these methods to a separate class, like `SwaggerConfiguration`

File: backend/src/main/java/com/gridstudentpractice/chatservice/controller/MessageRestController.java [hunk 2]
```diff
@@ -22,7 +22,7 @@
         return messageService.getMessages();
     }
 
-    @PostMapping
+    @PostMapping("/send-message")
```
Comment (Mantarus) [inline backend/src/main/java/com/gridstudentpractice/chatservice/controller/MessageRestController.java]: Again, you don't need "message" word here. just "send" will work just fine.

File: backend/src/main/java/com/gridstudentpractice/chatservice/controller/UserRestController.java [hunk 3]
```diff
@@ -19,7 +19,7 @@ public User getUser(@PathVariable String login) {
         return userService.getUser(login);
     }
 
-    @PostMapping
+    @PostMapping("/add-user")
```
Comment (Mantarus) [inline backend/src/main/java/com/gridstudentpractice/chatservice/controller/UserRestController.java]: Same here

File: backend/src/main/java/com/gridstudentpractice/chatservice/Application.java (hunk 1 above)
Comment (ktrof) [reply backend/src/main/java/com/gridstudentpractice/chatservice/Application.java]: done

File: backend/src/main/java/com/gridstudentpractice/chatservice/Application.java [hunk 4]
```diff
@@ -1,15 +1,27 @@
...
-    public static void main(String[] args) {
+    @Autowired
+    private SwaggerConfiguration swaggerConfiguration;
 
+    public static void main(String[] args) {
         SpringApplication.run(Application.class, args);
+    }
 
+    @Bean
+    public Docket setSwaggerConfiguration() {
```
Comment (Mantarus) [inline backend/src/main/java/com/gridstudentpractice/chatservice/Application.java]: You probably can just annotate `SwaggerConfiguration.setSwaggerConfiguration()` with `@Bean` and remove this method from `Application` class

File: backend/src/main/java/com/gridstudentpractice/chatservice/controller/UserRestController.java (hunk 3 above)
Comment (ktrof) [reply backend/src/main/java/com/gridstudentpractice/chatservice/controller/UserRestController.java]: done

File: backend/src/main/java/com/gridstudentpractice/chatservice/controller/MessageRestController.java (hunk 2 above)
Comment (ktrof) [reply backend/src/main/java/com/gridstudentpractice/chatservice/controller/MessageRestController.java]: done

File: backend/src/main/java/com/gridstudentpractice/chatservice/Application.java (hunk 4 above)
Comment (ktrof) [reply backend/src/main/java/com/gridstudentpractice/chatservice/Application.java]: done

**Pull request merged**